source/scripts/app.js

```diff
@@ -585,13 +612,16 @@ function bindCookbookCardButtons(card) {
 
   // get references to the buttons in the card
   let shadow = card.shadowRoot;
+  let title = shadow.querySelector(".title").innerHTML;
   let editButton = shadow.getElementById("edit");
   let removeButton = shadow.getElementById("remove");
   let openButton = shadow.getElementById("open");
 
   editButton.addEventListener("click", () => {
     // TODO set up cookbook editing
-    router.navigate("create-cookbook");
+    // Updates the CURRENT_COOKBOOK_TITLE
```

Comment:
Did you mean `COOKBOOK_TO_EDIT`?
```suggestion
    // Updates the COOKBOOK_TO_EDIT
```